post_office/test_settings.py

```diff
@@ -37,6 +37,8 @@
         'connection_tester': 'post_office.tests.test_mail.ConnectionTestingBackend',
     },
     'MAX_RETRIES': 2,
+    'MESSAGE_ID_ON_DELIVERY': False,
```

Comment:
No need to have this option, this just complicates things. If FQDN is specified, just generate this when email is initially created.

Reply:
I don't share this opinion. A variable shall be responsible for one type of information and only that one.
If we use `MESSAGE_ID_FQDN` then this variable turns on/off Message-ID generation *and* defines the right part of the Message-ID. This is kind of magic, tri-state behavior and should be avoided.

In addition to this, what if a user want to mimic the behavior of Django and use the server's DNS name?
What happens, if Django introduces its own overwritable "right" part of the Message-ID?

Therefore, if you really don't want two variables, let's drop `MESSAGE_ID_FQDN` instead of `MESSAGE_ID_ON_DELIVERY`.

Reply:
No, what I'm saying is that whether or not the ID is generated on delivery or when the email is created does not matter, since it's just a random uuid.